storage/store.go

```diff
@@ -448,7 +455,7 @@ func (s *DefaultStore) InsertExecutionSnapshot(
 		err := s.DataSetter.SetBytesWithVersion(
 			ctx,
 			s.KeyGenerator.Storage(LedgerStoreName),
-			[]byte(registerID.String()),
+			[]byte(registerID.Bytes()),
```

Comment:
here is breaking change with old data stores, but it is better than using String